packages/orbit-components/src/TextLink/TextLink.stories.js

```diff
@@ -138,6 +140,30 @@ export const Playground = (): React.Node => {
   );
 };
 
+export const TextLinkInText = (): React.Node => {
+  const type = select("type", Object.values(TEXT_TYPES), TEXT_TYPES.CRITICAL);
+  return (
+    <Text type={type}>
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nullam dapibus fermentum ipsum. Duis
+      ante orci, molestie vitae vehicula venenatis, tincidunt ac pede.{" "}
+      <TextLink onClick={action("onClick")} type={type}>
+        Etiam dui sem
+      </TextLink>
+      , fermentum vitae, sagittis id, malesuada in, quam. Vivamus luctus egestas leo. Integer
+      imperdiet lectus quis justo. Duis condimentum augue id magna semper rutrum. Quisque porta. Sed
+      elit dui, pellentesque a, faucibus vel, interdum nec, diam.
+    </Text>
+  );
+};
+
+TextLinkInText.story = {
+  name: "TextLink inside paragraph",
+  parameters: {
+    info:
+      "It inherits the visual style of the parent paragraph. Visit Orbit.Kiwi for more detailed guidelines.",
```

Comment:
Just to double check, link is supposed to be green only when the text is `primary`, right?

Reply:
Yes, because the TextLink is green/product only for the `primary` type.